server/model/taskModel.js

```diff
@@ -42,9 +45,8 @@ const TaskModel = {
             description: params.description,
             tasks: [],
             createdBy: params.createdBy,
-            tags: params.tags,
-            priority: params.priority
-
+            tags: params.tags != null? params.tags : [],
```

Comment:
please use strict equal |  not equal operators

Reply:
Done